src/js/select2/i18n/ka.js

```diff
@@ -0,0 +1,38 @@
+define(function () {
+  // Georgian
+  return {
+    errorLoading: function () {
+      return 'სია ცარიელია.';
```

Comment:
its valid translation, but because of Georgia is small country and not many people translates resources, automated translators aren't correct in most cases